client/state/selectors/is-gutenberg-opt-in-enabled.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Internal dependencies
+ */
+import { isEnabled } from 'config';
+import { isGutenbergSupported } from 'state/selectors/is-gutenberg-supported';
+
+export const isGutenbergOptInEnabled = ( state, siteId ) => {
+	return isGutenbergSupported( state, siteId ) && isEnabled( 'gutenberg/opt-in' );
```

Comment:
Should this also have the VIP check that's been removed from all of the other checks above?

Reply:
That check is already performed on the API, setting `is_gutenberg_supported_web` to false for VIP sites (D30746-code, `is_site_eligible_for_gutenberg`).

Since `isGutenbergSupported` reads the `is_gutenberg_supported_web` value returned by the API, we don't need to check again here.

Reply:
> Maybe this is being caught at various other points, but if there's ever a good reason for another "just to be sure" check, it's for VIP

Yeah, maybe I should add it back, just to have a 2nd check if for any reason the one in the API fails (or accidentally removed).

Reply:
80c7bd8